components/smalltalk/squeak4/squeak4.p5m

```diff
@@ -0,0 +1,49 @@
+file usr/lib/squeak/$(SQUEAK_REV)/so.UnixOSProcessPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.UnixOSProcessPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.RomePlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.RomePlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.FileCopyPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.FileCopyPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.DBusPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.DBusPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.UnicodePlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.UnicodePlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.Squeak3D group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.Squeak3D
+file usr/lib/squeak/$(SQUEAK_REV)/squeakvm group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/squeakvm
+file usr/lib/squeak/$(SQUEAK_REV)/so.ScratchPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.ScratchPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.ClipboardExtendedPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.ClipboardExtendedPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.SqueakSSL group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.SqueakSSL
+file usr/lib/squeak/$(SQUEAK_REV)/so.UUIDPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.UUIDPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/ckformat group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/ckformat
+file usr/lib/squeak/$(SQUEAK_REV)/so.GStreamerPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.GStreamerPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.vm-sound-null group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.vm-sound-null
+file usr/lib/squeak/$(SQUEAK_REV)/so.vm-display-null group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.vm-display-null
+file usr/lib/squeak/$(SQUEAK_REV)/so.SqueakFFIPrims group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.SqueakFFIPrims
+file usr/lib/squeak/$(SQUEAK_REV)/so.vm-display-custom group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.vm-display-custom
+file usr/lib/squeak/$(SQUEAK_REV)/so.XDisplayControlPlugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.XDisplayControlPlugin
+file usr/lib/squeak/$(SQUEAK_REV)/so.FT2Plugin group=bin mode=0555 owner=root path=usr/lib/squeak/$(SQUEAK_REV)/so.FT2Plugin
+file usr/bin/squeak.sh group=bin mode=0555 owner=root path=usr/bin/squeak.sh
+file usr/bin/squeak group=bin mode=0555 owner=root path=usr/bin/squeak
+
+file usr/share/man/man1/squeak.1 group=bin mode=0444 owner=root path=usr/share/man/man1/squeak.1
+file usr/share/doc/$(SQUEAK_VERSION)/LICENSE group=bin mode=0444 owner=root path=usr/share/doc/$(SQUEAK_VERSION)/LICENSE
+
```

Comment:
manifests/sample-manifest.p5m is missing from your commit.

Reply:
I've noticed that indeed many components have a manifests directory.  I'm not sure I fully understand what this directory is supposed to be (and yes, I've tried to read the doc/packaging.txt !).  In my case I guess I could add one by copying my squeak4.p5m manifest to manifests/sample-manifest.p5m ?

In any case thanks for reviewing and the suggestions.

Reply:
Whenever you create a new version you will run gmake sample-manifest again.
The diffs between the new sample-manifest.p5m and the old (git diff manifest/sample-manifest.p5m) will tell you what you'll need to change in your .p5m package file.
Otherwise you will have more work to find the differences and merge them into the p5m file.